packs/vehicles/karakoa.json

```diff
@@ -0,0 +1,122 @@
+{
+    "_id": "MKC2mjfMw9c9hdce",
+    "img": "systems/pf2e/icons/default-icons/vehicle.svg",
+    "items": [
+        {
+            "_id": "sT706wnF5RTbupUP",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Weakness 10 fire until broken",
+            "sort": 0,
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "interaction",
+                "description": {
+                    "value": ""
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [
+                    {
+                        "key": "Resistance",
```

Comment:
For other vehicles, we've used:
```
{
	"key": "Weakness",
	"predicate": [
		{
			"not": "self:condition:broken"
		}
	],
	"type": "fire",
	"value": 10
}
```

Reply:
The broken condition isn't added at BT for vehicles. This is what I initially tried but it doesn't work.